### ISSUE TITLE
Hidden from certain users??

### ISSUE THREAD

**Issue by szepeviktor** [easy-updates-manager/easy-updates-manager#22]
_Select which users to be excluded from the settings of this plugin. Default WordPress behavior will be used._

What does it mean? Will everyone understand it?

**Comment by ronalfy** [comment]
This is more for advanced users.

You can select a user which the plugin's settings will not apply.

Use-case, set up a client site, disable updates, etc.  Set up a developer account as an excluded user.

Developer can then log in and update the site as normal.

**Comment by szepeviktor** [comment]
Now I understand it.
Could you summarize it in the plugin also?

**Comment by ronalfy** [comment]
Sure.

**Comment by ronalfy** [comment]
See https://github.com/easy-updates-manager/easy-updates-manager/commit/04e6c17cc26f96449b9daf29766247fbb796484f
